lua/cfc_prop_restoration/core/sv/sv_init.lua

```diff
@@ -0,0 +1,78 @@
+local restorationFileName = "props_backup.json"
+local recent_disconnects = recent_disconnects or {}
```

Comment:
:snake: case why

Reply:
i think this variable name could be improved to reflect what it actually stores
maybe something like disconnectedDataExpirations ?